app/uk/gov/hmrc/thirdpartyapplication/models/ApplicationSearch.scala

```diff
@@ -133,18 +136,38 @@ case object AccessTypeFilter extends AccessTypeFilter {
   }
 }
 
+sealed trait LastUseDateFilter extends ApplicationSearchFilter
+case class LastUseBeforeDate(lastUseDate: DateTime) extends LastUseDateFilter
+case class LastUseAfterDate(lastUseDate: DateTime) extends LastUseDateFilter
+
+case object LastUseDateFilter extends LastUseDateFilter {
+  def apply(queryType: String, value: String): Option[LastUseDateFilter] = {
+    def asUTCTime(): DateTime = new DateTime(value.toLong, DateTimeZone.UTC)
```

Comment:
Would it be more user friendly to use an ISO format from JodaTime's ISODateTimeFormat (then you can use the parseDateTime method from the formatter)?